tests/ops/op_math/test_sprod.py

```diff
@@ -544,6 +544,18 @@ def test_simplify_nested_sprod_scalar_equal_to_1(self):
         assert simplified_op.data == final_op.data
         assert simplified_op.arithmetic_depth == final_op.arithmetic_depth
 
+    def test_simplify_with_sum_operator(self):
+        """Test the simplify method a scalar product of a Sum operator."""
+        sprod_op = s_prod(-3j, qml.op_sum(qml.PauliX(0), qml.PauliX(0)))
+        final_op = s_prod(-6j, qml.PauliX(0))
+        simplified_op = sprod_op.simplify()
+
+        assert isinstance(simplified_op, qml.ops.SProd)
+        assert simplified_op.name == final_op.name
```

Comment:
These might also be good cases to check the via the `repr` representation.
```suggestion
        assert simplified_op.name == final_op.name
        assert repr(simplified_op) == repr(final_op)
```